src/content/docs/en/community-resources/talks.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Talks, Interviews, and Streams
+description: Watch conference talks, interviews, and live streams about Astro!
+i18nReady: true
+---
+
+import Badge from '~/components/Badge.astro';
+
+Here are some of our favourite community videos to introduce you to Astro! 
+
+For more Astro content, check out [the Astro Blog](https://astro.build/blog/) or sign up for [the official Astro newsletter](https://astro.build/newsletter/signup) where we recap the best community content of each month.
+
+:::tip
+Subscribe to [the Astro Community Events Google Calendar](https://calendar.google.com/calendar/embed?src=c_g6re3u5gkl0mug3ktn1dibqhfk%40group.calendar.google.com) and never miss a live event!
+:::
+
+## About Astro
+
+- [Astro in 100 Seconds](https://www.youtube.com/watch?v=dsTXcSeAZq8) by Fireship
+- [Why I'll choose Astro (almost) every time in 2024](https://www.youtube.com/watch?v=kssIEqSJeMI) by Jason Lengstorf
+- [I tried Astro and I LOVE IT - 5 reasons you will too!](https://www.youtube.com/watch?v=wND4lSml31A) by James Q Quick
+- [What is Astro?](https://www.youtube.com/watch?v=oQDErNNzdwU) by Kodaps Academy
+- [Islands Architecture | Astro Islands Simply Explained](https://www.youtube.com/watch?v=kwqlP6Pe0SI) by ExcaliBear Codes
+- [Is Astro A Game Changer For Nuxt and Vue Developers?](https://www.youtube.com/watch?v=Sq7eKxPqfSo) by Program with Erik
+- [How Astro made my site 100x faster](https://www.youtube.com/watch?v=cOxA3kMYtkM) by James Perkins
+
+## Conference talks
+
+- [View Transitions: Fact vs. Fiction](https://www.youtube.com/watch?v=iT-3amHK7tA&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg&index=29)  by Fred Schott at ViteConf 2023
+- [Bringing React into your content sites with Astro](https://www.youtube.com/watch?v=Fouws8iKPKQ) by Matthew Phillips at Future Frontend 2023
+- [All Things Astro](https://www.youtube.com/watch?v=WKXHFIulZ9g) by Elian Van Cutsem, React Day Berlin 2023
+
+## Interviews and podcasts
+
+- [Does Astro make heavy JS frameworks obsolete?](https://www.youtube.com/watch?v=4Ohi9G-RCcE) with Fred K. Schott on Learn with Jason
+- [Coffee Chat about all things Astro](https://www.youtube.com/watch?v=3eyedOP83) with Matthew Phillips on WebStorm Webinar
```

Comment:
```suggestion
- [Coffee Chat about all things Astro](https://www.youtube.com/watch?v=3eyedOP83fw) with Matthew Phillips on WebStorm Webinar
```

Reply:
thanks for finding this out 🙌, youtube kept on pushing astrology related clips with this search